cs/json.texy

```diff
@@ -53,10 +51,14 @@ try {
 }
 \--
 
-Jak odeslat JSON z presenteru?
-==============================
 
-V metodě `action*()` nebo `render*()` zavoláme `$this->sendJson($payload);`, kde `$payload` obsahuje data, která chceme zakódovat do JSONu a odeslat.
+MVC - Presenter
+===============
+
+Jak odeslat JSON z presenteru? V `Presenter::action<>()` metodě zavoláme `$this->sendJson($payload)`, kde `$payload` jsou data, které chceme odeslat v JSON formátu.
```

Comment:
Proč ne render()? Resp. asi bych neřešil kde se to zavolá, je to úplně totiž jedno.